src/pages/Swap/LimitOrderBox/components/ConfirmLimitOrderModal/ConfirmationHeader.tsx

```diff
@@ -0,0 +1,103 @@
+import { CurrencyAmount, TokenAmount } from '@swapr/sdk'
+
+import styled from 'styled-components'
+
+import { ReactComponent as ArrowDown } from '../../../../../assets/images/bold-arrow-down.svg'
```

Comment:
Can we use react-feather icons here? Looks like we are adding an extra icon that we can reuse from react-feather :)

Reply:
Ohh didn't know it was feather icon will import it from there!